File: Python 3/CCC '14 J1 - Triangle Times.py
import sys
all_data = sys.stdin.read().split('\n')
a,b,c=int(all_data[0]),int(all_data[1]),int(all_data[2])
if a+b+c==180:
     if a==b==c:print("Equilateral")
     elif a==b or b==c or a==c:print("Isosceles")
     else:print("Scalene")
else:print("Error")

File: Java/CCC '14 J1 - Triangle Times.java
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class DMOJ {
 
 public static void main (String[] arg) throws Exception
 { 
  BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
  int a = Integer.parseInt(in.readLine()), b = Integer.parseInt(in.readLine()), c = Integer.parseInt(in.readLine());
  if(a+b+c==180)
  {
   if(a==b && b==c) System.out.print("Equilateral");
   else if (a==b || b==c || a==c) System.out.print("Isosceles");
   else System.out.print("Scalene");
  }
  else System.out.print("Error");
 }
}